### PR TITLE
CI Shouldn't Suck So Much!

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack 4bb81e8

File: .ci/appveyor.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ environment:
 cache:
   - "C:\\wheels"
   - "node_modules"
```
Comment (Adrianzatreanu) [inline .ci/appveyor.yml]: im not really sure, but i think theres an extra cache in the shortlog

Comment (sils1297) [inline .ci/appveyor.yml]: huh? What aere you saying?

**Comment by Adrianzatreanu** [comment]
the irony... ci failed ..

**Comment by sils1297** [comment]
because the cache doesn't exist yet, I restarted both builds if they pass that should be better ;)

**Comment by sils1297** [comment]
it has to pass once so it has the file cached

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**